Lib/_pyio.py

```diff
@@ -766,7 +766,7 @@ def truncate(self, pos=None):
 
     def flush(self):
         if self.closed:
-            raise ValueError("flush of closed file")
+            raise ValueError("flush on closed file")
```

Comment:
Hmm.  That's a good point.  'seek on' sounds right, as does 'tell on'.  'truncate on' doesn't. but to make it "right" I'd want it to be 'truncation of', which would lose the method name.

So, upon consideration consistency is better than "better English" in this case.  Keep the change.